<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_inventory_input_new</class>
 <widget class="QDialog" name="w_inventory_input_new">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="lds_toolbar_button" name="pushButton_new">
       <property name="text">
        <string>新增</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_del">
       <property name="text">
        <string>删除</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_save">
       <property name="text">
        <string>保存</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_audit">
       <property name="text">
        <string>审核</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton_blue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_bad">
       <property name="text">
        <string>作废</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton_blue</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_exit">
       <property name="text">
        <string>退出</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="outer_stylesheet" stdset="0">
      <string>frame_detail_show</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0,1">
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>入库类型</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_ch_sheetNo"/>
      </item>
      <item row="1" column="3">
       <widget class="lds_ComboBox" name="comboBox_vch_audit"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>审核人</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="lds_ComboBox" name="comboBox_vch_handle"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>供应商</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>单据号码</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>经手人</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="lds_ComboBox" name="comboBox_ch_operation"/>
      </item>
      <item row="2" column="1">
       <widget class="lds_ComboBox" name="comboBox_ch_providerNo"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>备注</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>税率</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="lds_spinbox" name="doubleSpinBox_int_tax">
        <property name="suffix">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QLineEdit" name="lineEdit_vch_memo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="lds_tableview_sum" name="tableView_detail"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_toolbar_button</class>
   <extends>QPushButton</extends>
   <header>lds_toolbar_button.h</header>
  </customwidget>
  <customwidget>
   <class>lds_ComboBox</class>
   <extends>QComboBox</extends>
   <header>lds_combobox.h</header>
  </customwidget>
  <customwidget>
   <class>lds_spinbox</class>
   <extends>QSpinBox</extends>
   <header>lds_spinbox.h</header>
  </customwidget>
  <customwidget>
   <class>lds_tableview_sum</class>
   <extends>QTableView</extends>
   <header>lds_tableview_sum.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
